src/odemis/driver/test/technolution_test.py

```diff
@@ -0,0 +1,1460 @@
+
+        self.assertEqual(min(X_descan_setpoints),
+                         numpy.floor(convertBits(descanner.scanOffset.value,
+                                                  numpy.array(descanner.scanOffset.range)[:, 1])[0]))
+        self.assertEqual(max(X_descan_setpoints),
+                         numpy.floor(convertBits(descanner.scanGain.value,
+                                                  numpy.array(descanner.scanGain.range)[:, 1])[0]))
+
+    def test_getYAcqSetpoints(self):
+        """
+        After each change the same properties of the testcases are checked, first the range of the values in the Y
+        setpouints is checked, then the length (total number of setpoints) is checked.
+        """
+        descanner = self.MirrorDescanner
+        mppc = self.MPPC
+        # Change values such that it is easy to follow the calculation
+        descanner.scanOffset.value = (0.09767299916075389, 0.09767299916075389)
+        descanner.scanGain.value = (0.646387426565957, 0.646387426565957)
+
+        # Check default values
+        Y_descan_setpoints = descanner.getYAcqSetpoints()
+        self.assertEqual(min(Y_descan_setpoints),
+                         math.floor(convertBits(descanner.scanOffset.value,
+                                                 numpy.array(descanner.scanOffset.range)[:, 1])[1])
+                         )
+        self.assertEqual(max(Y_descan_setpoints),
+                         math.floor(convertBits(descanner.scanGain.value[1],
+                                                 numpy.array(descanner.scanGain.range)[:, 1]))
+                         )
+        self.assertEqual(len(Y_descan_setpoints), mppc.cellCompleteResolution.value[1])
+
+        # Perform all check for different dwell_times (except default test)
+        for test_repetition in REPEAT_1000:
+            minimum_dwell_time = self.EBeamScanner.dwellTime.range[0]
+            random_dwell_time = numpy.round(numpy.random.random() * self.EBeamScanner.dwellTime.range[1], 6)
+            self.EBeamScanner.dwellTime.value = max(random_dwell_time, minimum_dwell_time)
+
+            # Check with changing gain
+            descanner.scanGain.value = (0.1, 0.7)  # only change Y value to a prime number
+            Y_descan_setpoints = descanner.getYAcqSetpoints()
+            self.assertEqual(min(Y_descan_setpoints),
+                             math.floor(convertBits(descanner.scanOffset.value,
+                                                     numpy.array(descanner.scanOffset.range)[:, 1])[1])
+                             )
+            self.assertEqual(max(Y_descan_setpoints),
+                             math.floor(convertBits(descanner.scanGain.value[1],
+                                                     numpy.array(descanner.scanGain.range)[:, 1]))
+                             )
+            self.assertEqual(len(Y_descan_setpoints), mppc.cellCompleteResolution.value[1])
+
+            # Change the cell_size
```

Comment:
What do you expect to happen? You increase/decrease the cell size -> what happens... add a comment.